libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -707,3 +707,27 @@ void AP_MotorsMatrix::thrust_compensation(void)
         _thrust_compensation_callback(_thrust_rpyt_out, AP_MOTORS_MAX_NUM_MOTORS);
     }
 }
+
+// set_output - spin a motor connected to the specified output channel.
```

Comment:
set_output -> output_test_num

Any reason not to put this method right after the other output_test_seq method?

Reply:
Done.